src/Blocks/BlockComparator.h

```diff
@@ -74,19 +74,21 @@ class cBlockComparatorHandler :
 
 	virtual bool CanBeAt(cChunkInterface & a_ChunkInterface, const Vector3i a_RelPos, const cChunk & a_Chunk) override
```

Comment:
It seems like this implements a very common behaviour -> requiresSolidBlockBelow. This should be extracted into a helper function or BlockHandler mixin.